jwcrypto/tests.py

```diff
@@ -786,6 +822,17 @@ def test_EdDsa_signing_and_verification(self):
             self.assertEqual(jws_verify.payload.decode('utf-8'),
                              curve_example['payload'])
 
+    def test_secp256k1_signing_and_verification(self):
+        key = jwk.JWK(**PrivateKeys_secp256k1['keys'][0])
+        payload = bytes(bytearray(A1_payload))
+        jws_test = jws.JWS(payload)
+        jws_test.add_signature(key, None, json_encode({"alg": "ES256K"}), None)
+        jws_test_serialization_compact = jws_test.serialize(compact=True)
+        jws_verify = jws.JWS()
+        jws_verify.deserialize(jws_test_serialization_compact)
+        jws_verify.verify(key.public())
+        self.assertEqual(jws_verify.payload, payload)
+
```

Comment:
This is good, but only guarantee we interoperate with ourselves.
Unfortunately I see no test vectors in the proposed draft, that should be addressed at IETF sending a comment they should provide official test vectors for both the secp256k1 curve as well as the RSASSA-PKCS1-5 algorithms.

Reply:
What do you think about merging this PR until RFC comes out? If the test vector comes out later, it would be better to modify it.

Reply:
Yes I am ok merging, this is why I asked to remove from default list.
Once it is final we can make any modification that is needed and add it to the default list.

Could you please squash the commits?
I will then merge.